src/storage.js

```diff
@@ -53,9 +60,116 @@ class LocalStorageStore {
  * It's not a problem if 'wallet:data' is not persisted, as this data is always loaded when we connect to the server.
  */
 class HybridStore {
+  static nonPrefixedKeyList = ['list-of-wallets'];
+
   constructor() {
     this.memStore = new hathorLib.MemoryStore();
     this.persistentStore = new LocalStorageStore();
+    this.prefix = '';
+    this.nonPrefixedKeys = new Set(HybridStore.nonPrefixedKeyList);
+  }
+
+  /*
+   * Upgrades an old version of the storage, which did not support multiple wallets
+   *
+   * @param {String} walletPrefix Prefix used to identify the wallet.
+   * @param {String} walletName Name used to identify the wallet.
+   *
+   * @memberof HybridStore
+   * @inner
+   */
+  upgradeStorage(walletPrefix, walletName) {
```

Comment:
Should we have a `storage:version` key to check if we have already upgraded this storage?